src/rpcblockchain.cpp

```diff
@@ -3668,25 +3669,19 @@ std::string SignBlockWithCPID(std::string sCPID, std::string sBlockHash)
 
 std::string GetPollContractByTitle(std::string objecttype, std::string title)
 {
-        for(map<string,string>::iterator ii=mvApplicationCache.begin(); ii!=mvApplicationCache.end(); ++ii) 
+    for(const auto& item : mvApplicationCache)
```

Comment:
These are repeating across all the functions. I will replace them all with an improved version of `GetListOf` in a later PR.